OrbitGrpcProtos/capture.proto

```diff
@@ -116,12 +118,38 @@ message GpuJob {
 }
 
 message ThreadName {
-  int32 pid = 1;
+  int32 pid = 1;  // Note that pid is currently not set.
   int32 tid = 2;
   string name = 3;
   uint64 timestamp_ns = 4;
 }
 
+message ThreadStateSlice {
+  int32 pid = 1;  // pid is currently not set as we don't have the information.
+  int32 tid = 2;
+
+  // Except for "running" (which is not a real state), these are the ones listed
```

Comment:
Yeah, so this is a matter of naming. In my naming with runnable I mean "can be run on a cpu" ("schedulable"), and with running "is currently run on the cpu" ("scheduled"). This is the naming that for example Perfetto, GAPID, Systrace use. This matches the common language.
But for the OS they are the same state. https://github.com/torvalds/linux/blob/master/fs/proc/array.c and https://man7.org/linux/man-pages/man5/proc.5.html call it "running", but https://www.man7.org/linux/man-pages/man1/ps.1.html#PROCESS_STATE_CODES calls it "running or runnable" (probably giving these words the same meaning as above). I guess that instead of saying that running is not a real state, I should say that running and runnable are the same state. I'll rephrase.